components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/service/GroupManagementProviderServiceImpl.java

```diff
@@ -69,7 +69,7 @@ public GroupManagementProviderServiceImpl() {
      */
     @Override
     public void createGroup(DeviceGroup deviceGroup, String defaultRole, String[] defaultPermissions)
-            throws GroupManagementException, GroupAlreadyEixistException {
+            throws GroupManagementException, GroupAlreadyExistException {
         if (deviceGroup == null) {
             throw new GroupManagementException("DeviceGroup cannot be null.", new NullPointerException());
```

Comment:
Nested exceptions are used to indicate root cause of the exception, then anyone can later get the type of wrapped exception by checking the nested one.